documentation/style-guide.rst

```diff
@@ -151,8 +151,8 @@ reading experience.
 Instead, these security concerns should be gathered into a dedicated
 "Security Considerations" section within the module's documentation, and
 cross-referenced from the documentation of affected interfaces with a note
-similar to ``"Please refer to the :ref:`security-considerations` section
-for important information on how to avoid common mistakes."``.
+similar to :samp:`"Please refer to the :ref:\`{security-considerations}\`
+section for important information on how to avoid common mistakes."`.
```

Comment:
This is a workaround for one of the two false positives.  It's semantically (and syntactically) valid, and solves the error reported by `sphinx-lint`.

It is rendered as
![image](https://user-images.githubusercontent.com/25624924/194411738-34d8f420-74f4-4e5e-b6f2-4a0b0736a5e8.png)
with the variable part in italic.